python/cugraph/link_analysis/hits.pxd

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
```

Comment:
date needs to be updated to 2020.  The date represents year when the file was created and if two years, the  modified year

Reply:
Fixed